src/modules/commander/Commander.cpp

```diff
@@ -1196,37 +1216,39 @@ Commander::handle_command(const vehicle_command_s &cmd)
 			break;
 		}
 
-	case vehicle_command_s::VEHICLE_CMD_START_RX_PAIR:
```

Comment:
just alphabetized this list along with the addition of `vehicle_command_s::VEHICLE_CMD_SET_GPS_GLOBAL_ORIGIN`